src/ExportMap.js

```diff
@@ -344,14 +345,43 @@ ExportMap.parse = function (path, content, context) {
   var m = new ExportMap(path)
 
   try {
-    var ast = parse(path, content, context)
+    var { ast, visitorKeys } = parse(path, content, context)
   } catch (err) {
-    log('parse error:', path, err)
     m.errors.push(err)
     return m // can't continue
   }
 
-  if (!unambiguous.isModule(ast)) return null
+  let hasDynamicImports = false
+
+  visit(ast, visitorKeys, {
+    CallExpression(node) {
+      if (node.callee.type === 'Import') {
+        hasDynamicImports = true
+        const firstArgument = node.arguments[0]
```

Comment:
```suggestion
    ImportExpression(node) {
      hasDynamicImports = true
      const source = node.source
        
      // replace `firstArgument` with `source` below
```

Reply:
We'd need to keep both; `ImportExpression` is the newer form, but older parsers still have a `CallExpression` of callee type Import.